src/pages/candidate-info/candidate-info.html

```diff
@@ -6,77 +6,77 @@
 -->
 
 <ion-header>
-  <page-header></page-header>
+    <page-header></page-header>
```

Comment:
hym, should we agreed on tab vs spaces approach. I'm personally fun of 2 spaces but if everybody loves tabs we could start using tabs. 

Tbh I would like to our code convention be consistent across all our subprojects

Reply:
@zielinskikamil - the prettier package will help enforce the code styles (tabs, semicolons etc), we just need to evolve it until we are happy with it.